provers/blevm/README.md

```diff
@@ -1 +1,38 @@
 # blevm
+
+blevm is a service that creates zero-knowledge proofs of EVM state transitions.
```

Comment:
we should probably add more details but you don't have to do it in this PR. we can export this comment as an issue.